fdbbackup/FileDecoder.actor.cpp

```diff
@@ -448,15 +449,15 @@ ACTOR Future<Void> decode_logs(DecodeParams params) {
 	for (; i < logs.size(); i++) {
 		if (logs[i].fileSize == 0) continue;
 
-		state DecodeProgress progress(logs[i], left);
+		state DecodeProgress progress(logs[i], std::move(left));
 		wait(progress.openFile(container));
 		while (!progress.finished()) {
 			VersionedMutations vms = wait(progress.getNextBatch());
 			for (const auto& m : vms.mutations) {
 				std::cout << vms.version << " " << m.toString() << "\n";
 			}
 		}
-		left = progress.getUnfinishedBuffer();
+		left = std::move(progress).getUnfinishedBuffer();
```

Comment:
```suggestion
		left = std::move(progress.getUnfinishedBuffer());
```

Reply:
This was intentional, to clearly indicate that `progress` can't be used again, now that its contents have been moved from. `getUnfinishedBuffer()` already returns an rvalue reference, so it doesn't need to be moved from.